keep-runtime/src/workload.rs

```diff
@@ -93,7 +89,9 @@ pub(crate) mod test {
 
     #[test]
     fn workload_run_return_1() {
-        let bytes = include_bytes!(concat!(env!("OUT_DIR"), "/fixtures/return_1.wasm")).to_vec();
+        let path = std::path::Path::new("fixtures").join("return_1.wat");
+
+        let bytes = wat::parse_file(&path).unwrap();
```

Comment:
It seems that something has messed up during the rebase - the new code is actually the old code before https://github.com/enarx/enarx/commit/2d451232578fbd8be65777e5bea7d55ac95766b5

Reply:
I think I've fixed this.  I'm not reading from a file: this accepts bytes from the main.rs file (received over an HTTPS connection).